src/System.Private.CoreLib/shared/System/Globalization/TextInfo.cs

```diff
@@ -602,7 +603,7 @@ private void PopulateIsAsciiCasingSameAsInvariant()
         /// </summary>
         public bool IsRightToLeft => _cultureData.IsRightToLeft;
 
-        public override bool Equals(object obj)
```

Comment:
Github won't let me comment but s_Invariant, listSeparator and others can actually be null, no? They're laziy initialized.

Reply:
s_invariant true - must have hit the same bug @stephentoub hit here - it didn't complain about it. listSeparator already is nullable. Rest seem that it can be non-nullable.. any specific in mind?